modules/compliance-review.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-review_{context}"]
+= Reviewing a remediation
+
+When reviewing a remediation, it is prudent to review both the `ComplianceRemediation` object itself, but also the `ComplianceCheckResult` object that owns the remediation. The `ComplianceCheckResult` object contains human-readable descriptions of what the check does, what is the hardening trying to prevent, as well as other metadata like the severity and the associated security controls. The `ComplianceRemediation` object represents a way to fix the problem described in the `ComplianceCheckResult`.
+
+Below is an example of a check and a remediation called `sysctl-net-ipv4-conf-all-accept-redirects`. This example is redacted to only show spec and status and omits metadata:
+
+[source,yaml]
+----
+  spec:
+    apply: false
+    current:
+  	object:
+    	apiVersion: machineconfiguration.openshift.io/v1
+    	kind: MachineConfig
+    	spec:
+      	config:
+        	ignition:
+          		version: 2.2.0
+        	storage:
+          	files:
+       		   	- contents:
+             	 	source: data:,net.ipv4.conf.all.accept_redirects%3D0
+            		filesystem: root
+            		mode: 420
+            		path: /etc/sysctl.d/75-sysctl_net_ipv4_conf_all_accept_redirects.conf
+    outdated: {}
+  status:
+    applicationState: NotApplied
```

Comment:
for yaml, I suggest only using spaces (2 spaces to be exact). tabs end up quite confusing and will cause issues with parsing too.

Reply:
Looks like atom is inserting it's own tabs, which is why this looks so weird. I've fixed with actual spaces. Thanks!